asfktz/autodll-webpack-plugin
# add watch option for cache validation

1st pass of providing [watch mode](https://github.com/asfktz/autodll-webpack-plugin/issues/7).
Right now this PR only hash the files and folders in watch settings to calculate the value of `isCacheValid`

To test:

1. cd to `examples/watch`
2. run npm start
3. ctrl + c
4. run npm start again to verify that cache is working
5. modify package.json
6. npm start -> cache is invalid, DLL is rebuilt

TODOS:
<del>- [ ] Watch for the changes in these files and folder and rebuild without restarting devServer </del>  holding this one for future development
